bin/src/hickory-dns.rs

```diff
@@ -273,6 +273,14 @@ async fn load_zone(
 #[derive(Debug, Parser)]
 #[clap(name = "Hickory DNS named server", version, about)]
 struct Cli {
+    /// Test validation of configuration files
+    #[clap(long = "validate", conflicts_with = "quiet")]
```

Comment:
The validation message prints out an `info!` message on return.

Reply:
IMO the process exit value should also reflect the status (that is, `main()` should yield `Err` or `ExitCode::FAILURE`) and it would still be useful even if `--quiet`.